ui/src/components/query/builder/edit/dialogs/SaveFeatureDialog.vue

```diff
@@ -0,0 +1,208 @@
+        <small class="p-error" id="text-error">{{ errors.iri || "&nbsp;" }}</small>
+      </div>
+
+      <div class="flex flex-column gap-2" id="save-feature-scheme-iri">
+        <span id="save-feature-scheme-header">Scheme</span>
+        <Dropdown
+          id="scheme"
+          v-bind="scheme"
+          type="text"
+          :class="{ 'p-invalid': errors.scheme }"
+          aria-describedby="text-error"
+          :options="schemeOptions"
+          option-label="name"
+          option-value="@id"
+          placeholder="Scheme"
+          class="flex-1"
+        />
+        <small class="p-error flex-1" id="text-error">{{ errors.scheme || "&nbsp;" }}</small>
+      </div>
+
+      <div class="flex flex-column gap-2" id="save-feature-name">
+        <span id="save-feature-name-header">Name</span>
+        <InputText id="name" v-bind="name" type="text" :class="{ 'p-invalid': errors.name }" aria-describedby="text-error" />
+        <small class="p-error" id="text-error">{{ errors.name || "&nbsp;" }}</small>
+      </div>
+
+      <span id="definition-title">Definition</span>
+      <div class="save-feature-query-display"><QueryDisplay :definition="definition" :hide-sql-button="true" /></div>
+    </form>
+
+    <template #footer>
+      <Button label="Discard" severity="secondary" @click="onDiscard" text />
+      <Button type="submit" text label="Submit" @click="onSubmit" :loading="loading" />
+    </template>
+  </Dialog>
+</template>
+
+<script setup lang="ts">
+import { ComputedRef, Ref, computed, onMounted, ref, watch } from "vue";
+import { useForm } from "vee-validate";
+import { isArrayHasLength, isObjectHasKeys } from "@im-library/helpers/DataTypeCheckers";
+import { Node, TTIriRef, Match } from "@im-library/interfaces/AutoGen";
+import { EntityService, FilerService, FunctionService, QueryService } from "@/services";
+import { IM, RDF, RDFS } from "@im-library/vocabulary";
+import { useToast } from "primevue/usetoast";
+import * as yup from "yup";
+import QueryDisplay from "@/components/directory/viewer/QueryDisplay.vue";
+
+interface Props {
+  showDialog: boolean;
+  feature: Match;
+}
+
+const props = defineProps<Props>();
+
+const emit = defineEmits({
+  "update:showDialog": payload => typeof payload === "boolean"
+});
+const schema = yup.object({
+  scheme: yup
+    .string()
+    .required()
+    .label("Scheme")
+    .default(IM.NAMESPACE + "M_"),
+  iri: yup
+    .string()
+    .required()
+    .test("isValidIri", "Iri exists", async () => await isValidIri())
+    .label("Iri"),
+  name: yup.string().required().label("Name"),
+  type: yup.string().required().label("Type").default(IM.FEATURE)
+});
+
+const toast = useToast();
+
+const { defineComponentBinds, handleSubmit, resetForm, errors, setFieldValue } = useForm({
+  validationSchema: schema
+});
+
+const fullIri: ComputedRef<string> = computed(() => `${scheme.value.modelValue ?? ""}${iri.value.modelValue ?? ""}`);
```

Comment:
do you need backtick string variables?
=> scheme.value.modelValue ?? "" + iri.value.modelValue ?? ""
looks like that would work...